app/pulp/app/apps.py

```diff
@@ -75,4 +75,4 @@ class PulpAppConfig(PulpPluginAppConfig):
     # The app label to be used when creating tables, registering models,
     # referencing this app with manage.py, etc. This cannot contain a dot,
     # so for brevity's sake it's just "pulp", rather than e.g. "pulp_app".
-    label = 'pulp'
+    label = 'pulp_app'
```

Comment:
You've made a liar out of the preceding comment. It should now read something along the lines of "This cannot contain a dot and must not conflict with the name of a package containing a Django app, so it should be "pulp_app" rather than "pulp".